tests/auth/test_generic_auth.py

```diff
@@ -187,8 +187,8 @@ def test_update_account_when_no_server_provided(db):
     assert acc_smtp_port == smtp_port
 
 
-@pytest.mark.networkrequired
-def test_double_auth(db):
+@pytest.mark.usefixtures('mock_smtp_get_connection')
```

Comment:
We had a conversation the other day where we decided fixtures the test actually manipulates should be arguments whereas fixtures that are only added to have some sort of other effect should be provided via a decorator.